setup.py

```diff
@@ -112,9 +85,7 @@ def run(self):
     },
 
     install_requires=[
-        # Any changes here must be reflected in ``python-flocker.spec.in`` so
-        # that RPM dependencies match.
-        # They must also be reflected in the yumdownloader lines in
+        # Any changes here must be reflected in the yumdownloader lines in
```

Comment:
We shouldn't need any ``yumdownloader`` lines for python dependencies, for omnibus builds. Only non-python dependencies.